apps/mochi-web/pages/changelog.tsx

```diff
@@ -29,36 +31,59 @@ export const getStaticProps: GetStaticProps<Props> = async () => {
   }
 }
 
-const Heading = ({ children }: { children: React.ReactNode }) => {
-  return <p className="text-2xl font-medium">{children}</p>
+const Heading = ({
+  children,
+  level,
+}: {
+  children: React.ReactNode
+  level: number
+}) => {
+  return (
+    <Typography
+      level={`h${level}` as TypographyProps['level']}
+      fontWeight="md"
+      className={clsx('py-8 overflow-visible', {
+        '!text-[32px]': level === 1,
+        '!text-2xl': level === 2,
+        '!text-xl': level > 2,
+      })}
+    >
+      {children}
+    </Typography>
+  )
+}
+
+const UnorderedList = ({ children }: { children: React.ReactNode }) => {
+  return <ul className="list-dashed">{children}</ul>
 }
 
 const Paragraph = ({ children }: { children: React.ReactNode }) => {
-  return <span className="mb-8 font-normal">{children}</span>
+  return (
+    <Typography level="p1" className="text-xl font-normal pb-8">
+      {children}
+    </Typography>
+  )
 }
 
 const Image = (props: any) => {
   return (
     <NativeImage
-      className="-mt-5 w-[70%] mx-auto rounded-lg"
+      className="w-full mx-auto rounded-lg"
       src={props.src}
       alt={props.alt || ''}
     />
   )
 }
 
 const ChangelogItem = ({ name, content }: Page) => (
-  <div className="gap-9 mb-16 lg:flex">
-    <div className="inline-block relative flex-shrink-0 mb-5 lg:pt-2">
-      <div className="top-36 lg:sticky">
-        <div className="bg-white rounded-xl border border-gray-200">
-          <div className="relative px-4 font-semibold leading-9 text-center text-gray-600 lg:px-5">
-            {name}
-          </div>
-        </div>
+  <div className="gap-8 mb-24 md:flex justify-center">
+    <div className="inline-block relative w-full md:w-[176px] flex-shrink-0 mb-12 md:mb-0">
+      <div className="top-24 md:sticky flex flex-row md:flex-col gap-4 md:gap-2 items-center md:items-start">
+        <Badge className="w-max !text-base !rounded-md !px-4">v1.52.0</Badge>
```

Comment:
Is there no default variant of the badge to match the design? If that's the case, we should either fix the design or fix the badge itself.